### DISCUSSION TITLE
Use a full top-bar for the Sphinx Book Theme

### DISCUSSION THREAD

**Discussion by choldgraf** [executablebooks/meta#296] [Ideas]
I was inspired by @drdrij's work on the QuantEcon theme, and wanted to know what it'd look to have a "full-width topbar" for the Sphinx Book Theme (instead of the sidebar-based structure). Here's how it looks with pretty minimal hacking:

![image](https://user-images.githubusercontent.com/1839645/112755949-d5848e80-8f97-11eb-9912-382dc0b0bf3c.png)

You could also imagine adding extra UI elements (optionally) like a topbar-based logo:

![image](https://user-images.githubusercontent.com/1839645/112739816-54dc7880-8f2c-11eb-89e4-d3430cb550d6.png)

I'm curious if folks think that this is a reasonable design direction to pursue. Some things I like about it:

- There is less dead space in the top left
- This would fit horizontal logos more cleanly
- I'm not sure why but the page layout feels less awkward to me this way
- There's pretty much the same interface on mobile or widescreen
- The "collapse sidebar" button doesn't move when the sidebar moves

stuff that I think needs improving

- The topbar is too scrunched
- Still not sure how to handle the borders between sections of the page

Would love to know what the @executablebooks/ebpteam thinks about this (in particular perhaps @pradyunsg and @drdrij and @chrisjsewell , who I know are interested in improving the theme look and feel)

would also be curious what @kirstiej and @sgibson91 think - maybe some of this UI change could make it easier to solve subsets of book structure? E.g., you could imagine a dropdown in the top-left that would let you select different sub-sections of the book? Maybe I am getting ahead of myself here 😅

You can also checkout a minimal PR that implements this here: https://github.com/executablebooks/sphinx-book-theme/pull/317

**Comment by DrDrij** [comment]
@choldgraf Neat! I really like it.

Users will need to watch out for logos that don't scale well. But as long as the option is there to have a larger logo in the left column.

Further - I'd make a suggestion that the logo can be displayed in full on the book index page and a small version in the toolbar as you have it Chris.
